pkg/grpc/api.go

```diff
@@ -1499,7 +1503,14 @@ func (a *api) SubscribeConfigurationAlpha1(request *runtimev1pb.SubscribeConfigu
 	return nil
 }
 
-func (a *api) UnsubscribeConfigurationAlpha1(ctx context.Context, request *runtimev1pb.UnsubscribeConfigurationRequest) (*runtimev1pb.UnsubscribeConfigurationResponse, error) {
+func (a *api) SubscribeConfigurationAlpha1(request *runtimev1pb.SubscribeConfigurationRequest, configurationServer runtimev1pb.Dapr_SubscribeConfigurationAlpha1Server) error { //nolint:nosnakecase
+	// Currently, the alpha API has the same interface as the stable API, so we add `var _ ...` to guarantee this by the compiler.
+	// Once compile fails, `configurationServer.(runtimev1pb.Dapr_SubscribeConfigurationServer)` should be changed then, and this line can be deleted.
+	var _ runtimev1pb.Dapr_SubscribeConfigurationServer = runtimev1pb.Dapr_SubscribeConfigurationAlpha1Server(nil)
```

Comment:
@shivamkm07 This var _ is not executed until this API is called. 
This is not a compile time check. 
Also add a TODO for removing Alpha1 API.